bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v2/PageImpl.java

```diff
@@ -88,6 +92,11 @@
      */
     public static final String PN_MAIN_CONTENT_SELECTOR_PROP = "mainContentSelector";
 
+    /**
+     * Property name of the style property that enables/disables rendering of the alternate language links
+     */
+    public static final String PN_RENDER_ALTERNATE_LANGUAGES = "renderAlternateLanguageLinks";
```

Comment:
I know we are not very consistent with this (see above) but I prefer we have the constants that describe properties as members of the model interface. Also, in this case, the naming should probably begin with `PN_STYLE_*`

Reply:
Will do that